server/models.py

```diff
@@ -54,6 +111,10 @@ def rows(self):
             for _, g in itertools.groupby(seats, lambda seat: seat.row)
         ]
 
+    def __repr__(self):
+        return '<Room {}>'.format(self.name)
+
+
 class Seat(db.Model):
     __tablename__ = 'seats'
     id = db.Column(db.Integer, primary_key=True)
```

Comment:
Not for now, but we should consider how we can use rooms without a seat map.

Reply:
what for?

like small dsp rooms where exact seat map does not matter (only seat count matters)?